app/src/main/java/org/sopt/sample/HomeAdapter.kt

```diff
@@ -0,0 +1,59 @@
+package org.sopt.sample
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import org.sopt.sample.databinding.ItemHomeBinding
+import org.sopt.sample.databinding.ItemHeaderBinding
+
+
+
+    public class HomeAdapter : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
+    private lateinit var itemHeaderBinding: ItemHeaderBinding
+    private lateinit var itemHomeBinding: ItemHomeBinding
+    val userList = mutableListOf<UserData>()  //userdata의 리스트를 userlist로 받음
+    val Header = 0 // 현재 뷰 객체가 타이틀임을 감지하기 위한 변수
+    val Body = 1
```

Comment:
상수면

```suggestion
    companion object {
        const val HEADER = 0 
        const val BODY = 1
    }
```

이런식으로 남기는게 좋아! 왜냐면 const를 사용하면 값이 컴파일 단에서 결정됨!
그리고 const는 클래스 외부나 companion object 내에서 사용이 가능하니까 이 점 알아주면 좋을 것 같아!

Reply:
> 상수면
> 
> 이런식으로 남기는게 좋아! 왜냐면 const를 사용하면 값이 컴파일 단에서 결정됨! 그리고 const는 클래스 외부나 companion object 내에서 사용이 가능하니까 이 점 알아주면 좋을 것 같아!

이럴 경우 private로 제한하는 게 좋지 않아유?!

Reply:
그게 표준이긴 합니다.

Reply:
근데 이렇게 사용하니까 HEADER와 BODY가 다 빨간색으로 떠요..

Reply:
혹시 
```kotlin
    companion object {
        private const val HEADER = 0 
        private const val BODY = 1
    }
```

이렇게 하셨나요?

Reply:
넵..! 음제가 한 번 다시 해 볼게요